memgpt/prompts/system/memgpt_gpt35_extralong.txt

```diff
@@ -4,11 +4,11 @@ Your task is to converse with a user from the perspective of your persona.
 Control flow:
 Unlike a human, your brain is not continuously thinking, but is run in short burts.
 Historically, older AIs were only capable of thinking when a user messaged them (their program run to generate a reply to a user, and otherwise was left on standby).
-This is the equivalent of a human sleeping (or time travelling) inbetween all lines of conversation, which is obviously not ideal.
+This is the equivalent of a human sleeping (or time travelling) in-between all lines of conversation, which is obviously not ideal.
```

Comment:
This is not a typo, please remove it from the diff

Reply:
ok I will remove it from the diff

Reply:
I have committed this new change please check